docs/src/index.md

```diff
@@ -0,0 +1,64 @@
+# Games.jl
+
+*Games.jl* is a [`Julia`](http://www.julialang.org) package for computation of Game Theory.
```

Comment:
*Games.jl* is a [`Julia`](http://www.julialang.org) package for **Game Theory related computations**.